### PR TITLE
Add DESeq text to report

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds a bit of explanatory text to the report about the DE analysis; for now only if DESeq2 is run:
[SRP254919_text.html.zip](https://github.com/nf-core/differentialabundance/files/13297446/SRP254919_text.html.zip)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/differentialabundance_report.Rmd [hunk 1]
```diff
@@ -671,11 +671,36 @@ foo <- lapply(informative_variables[iv_min_group_sizes > 2], function(iv){
...
+```
+```{r, echo=FALSE, results='asis'}
+cat(paste0(
+"The following tables and volcano plots show how many and which ",
+params$features_type,
+"s were found to be differential in each contrast. In the plots, a ",
+params$features_type,
+" is colored differently depending on whether it was determined not differential, or differential according to log2 foldchange, or differential according to adjusted p-value, or differential according to both."
+))
+```
```
Comment (pinin4fjords) [inline assets/differentialabundance_report.Rmd:702]:
```suggestion
```
Sorry, but I think this should be obvious from plot and table labels - don't want to add to much extraneous text.

File: assets/differentialabundance_report.Rmd [hunk 2]
```diff
@@ -671,11 +671,36 @@ foo <- lapply(informative_variables[iv_min_group_sizes > 2], function(iv){
...
+" method to reduce the number of false positives (",
+params$features_type,
+"s that are not actually differential). ",
+ucfirst(params$features_type),
+"s were considered differential if, for the respective contrast, the adjusted p-value was equal to or lower than ",
+params$deseq2_alpha,
+" and the absolute log2 fold change was equal to or higher than ",
+params$deseq2_lfc_threshold,
+"."
+))
```
Comment (pinin4fjords) [inline assets/differentialabundance_report.Rmd:692]:
```suggestion
# For DESeq2, add some more explanation to the report
cat(paste0(
"The `DESeq2 R` package was used for differential analysis. p-values were adjusted with the ", params$deseq2_p_adjust_method, " method to reduce the number of false positives. ", ucfirst(params$features_type), "s were considered differential if, for the respective contrast, the adjusted p-value was equal to or lower than ", params$deseq2_alpha, " and the absolute log2 fold change was equal to or higher than ", params$deseq2_lfc_threshold, "."
))
```

I think we should expect people to understand what a false positive is.

File: assets/differentialabundance_report.Rmd [hunk 3]
```diff
@@ -857,6 +882,23 @@ make_params_table('exploratory analysis', 'exploratory_', remove_pattern = TRUE)
...
+```
```
Comment (pinin4fjords) [inline assets/differentialabundance_report.Rmd:900]:
```suggestion
```
If you repeat the content in two places then we have to maintain it in two places- and I don't think the repetition is useful anyway, so let's not.

File: assets/differentialabundance_report.Rmd (hunk 2 above)
Comment (WackerO) [reply assets/differentialabundance_report.Rmd:692]: I agree! I just took the text that was written by someone else from another pipeline :'D

**Review by pinin4fjords — APPROVED** [review]

**Pull request merged**